game/game.js

```diff
@@ -413,32 +418,29 @@ function canPlay(baseGame, username){
 }
 
 /**
- * vérifie que tout les joueurs ont trouvés leur scénario
- *  @param {object} baseGame Instance de jeu
+ * vérifie que tout les joueurs ont trouvé leur scénario
+ *  @param {object} game Instance courrante de jeu
  * @returns {boolean} goForTheFinal indique si le scenario final doit être lancé ou non
  */
-function allScenariosFinded (baseGame){
-    let goForTheFinal = true ;
-    baseGame.players.forEach(function(player){
-        if(!scenarioFind(baseGame,player.username)){
-            goForTheFinal = false;
+function areAllScenariosFound (game){
+    let canGoForTheFinal = true ;
+    game.players.forEach(function(player){
```

Comment:
This can be written like: `game.players.forEach((player)=>{`

Reply:
Using forEach is already nice, but you can even use [some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) which would fit better.